plugins/jobs/server/models/job.py

```diff
@@ -52,19 +56,33 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
-    def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
+    def list(self, user, types=None, statuses=None,
```

Comment:
I didn't pay close enough attention here either.  You've changed the default for the `user` param, which breaks backwards compatibility.  Can you reset it to `None`? Is there a reason you needed to use the string value `none` instead of the Python `None`?

It is possible you did this because of review feedback I gave you before, I apologize if I've created any confusion.